src/fsharp/service/FSharpCheckerResults.fs

```diff
@@ -1980,7 +1980,9 @@ type FSharpCheckFileResults
                    showImperativeTyparAnnotations=true
                    showHiddenMembers=true
                    showObsoleteMembers=true
-                   showAttributes=true }
+                   showAttributes=true
+                   shrinkOverloads=false
```

Comment:
Does this impact tooltips? We shrink overloads for those

Reply:
No, this is a separate pass to generating text versus tooltips.